tests/compose-rootfs/Containerfile

```diff
@@ -2,27 +2,34 @@
 FROM quay.io/centos/centos:stream10 as repos
 
 # You must run this build with `-v /path/to/rpm-ostree:/run/build/rpm-ostree:ro`
-FROM quay.io/fedora/fedora:41 as builder
+FROM quay.io/fedora/fedora-bootc:41 as builder
 RUN <<EORUN
 set -xeuo pipefail
-# Install our dependencies
-dnf -y install rpm-ostree selinux-policy-targeted sqlite
 # Our goal here though is to test the updated rpm-ostree binary.
 # Right now there are a very few things that live outside the binary
 # like rpm-ostree-0-integration.conf, but we should probably move those in.
+# nocache 0228
```

Comment:
And yeah the fact that dynamic bind mount changes aren't accounted for in podman build caching is soo painful. [So true](https://news.ycombinator.com/item?id=7037530)